scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/services/TypescriptCodeGenerationService.java

```diff
@@ -86,11 +86,35 @@ private static String generateActivityTemplateConstructors(final Iterable<Activi
     final var result = new ArrayList<String>();
     result.add("const ActivityTemplateConstructors = {");
     for (final var activityTypeCode : activityTypeCodes) {
-      result.add(indent("%s: function %sConstructor(args: {".formatted(activityTypeCode.activityTypeName(), activityTypeCode.activityTypeName())));
-      result.add(indent(indent(generateActivityArgumentTypes(activityTypeCode.parameterTypes()))));
-      result.add(indent("}): %s {".formatted(activityTypeCode.activityTypeName())));
-      result.add(indent(indent("return { activityType: ActivityType.%s, args };".formatted(activityTypeCode.activityTypeName()))));
-      result.add(indent("},"));
+      if(activityTypeCode.parameterTypes().size() == 0) {
+        /*
+        Relax: function RelaxConstructor(): Relax {
```

Comment:
This comment looks like it was helpful when writing this code, but I'm worried it will mostly be confusing in the future. For one, it's already out of date (the returned dictionary is missing the `args` key) and it will only get more out of date as we continue to make changes to this code. I suggest removing these comments, but making sure that the `TypescriptCodeGenerationTest` has examples of activities with and without parameters.

Reply:
I removed it!